src/systems/systems/src/shared/random.rs

```diff
@@ -20,6 +20,16 @@ mod wasm {
         // SAFETY: `RAND` was initialized above.
         unsafe { (*RAND.as_mut_ptr()).gen() }
     }
+
+    pub fn random_from_range(from: f64, to: f64) -> f64 {
+        // SAFETY: WASM is single-threaded, and we're not passing references to `RAND` around.
+        RAND_INIT.call_once(|| unsafe {
+            RAND = MaybeUninit::new(SmallRng::from_entropy());
+        });
+
+        // SAFETY: `RAND` was initialized above.
+        unsafe { (*RAND.as_mut_ptr()).gen_range(from..to) }
```

Comment:
Oh I thought the call_once was indeed doing nothing after first call?

Reply:
@SparkyPotato Okay. I just read the documentation and indeed, duplicating the `call_once` code doesn't matter, as the `Once` type ensures it will really only be called once.

Your other comment makes sense to me too! 😀